audio/filter/af_scaletempo2_internals.c

```diff
@@ -104,6 +104,10 @@ static float multi_channel_similarity_measure(
     return similarity_measure;
 }
 
+#if HAVE_VECTOR
```

Comment:
<pre>
../audio/filter/af_scaletempo2_internals.c:107:5: Warnung: »HAVE_VECTOR« ist nicht definiert, wird zu 0 ausgewertet [-Wundef]
  107 | #if HAVE_VECTOR
      |     ^~~~~~~~~~~

</pre>

Reply:
Did you run a `./waf configure` beforehand?

Reply:
All the `HAVE_*` values are defined at `config.h`, but `config.h` is not included in this file, and it's not obvious that it's included from other headers here.

Reply:
Fixed.